_pages/emergency-recovery/index.md

```diff
@@ -3,45 +3,48 @@ title: Emergency Recovery
 ---
 
 By default, you sign your `2-of-3` transactions on your `2` hardware wallets (Cobo Vault and Coldcard) that [you previously configured](/setup-wallets/).
-If either device is damaged/destroyed, you should replace it with a new one and load the replacement device with the same seed phrase that [you previously backed up](/backup-wallet/seeds).
+If one device is damaged/destroyed/lost, that means that you still have the other 1 good hardware wallet to sign with, and you still have 2 other seeds backed up (1. the destroyed wallet's [backup seed](/backup-wallet/seeds), and 2. your 3rd seed which is on the [paper wallet](/setup-wallets/paper)) to choose from to spend your coins.
+
+In addition, this page might be useful if you want to test that you are able to successfully sign a transaction with your 3rd seed from the paper wallet, before you deposit any significant funds into the multisig wallet. That way you can be sure that all 3 seeds do in fact work. Its always a good idea to test your backups.
```

Comment:
hi, which "page" is meant - the page where the 3rd paper wallet seed is written to? But in general: very good idea to test the 3rd seed! One could also load it to a 3rd HWW, if there is any, and check if a signature is possible.

Reply:
The "page" meant is the page they are currently reading